python/tests/test_random_forest.py

```diff
@@ -700,17 +700,32 @@ def test_fit_multiple_in_single_pass(
         initial_rf = rf.copy()
 
         param_maps = [
-            {rf.maxDepth: 3, rf.maxBins: 16},
-            {rf.maxDepth: 3, rf.maxBins: 42},
-            {rf.maxDepth: 6, rf.maxBins: 16},
-            {rf.maxDepth: 6, rf.maxBins: 42},
+            {rf.maxDepth: 3, rf.maxBins: 3, rf.numTrees: 5},
```

Comment:
Would be good to vary featuresSubsetStrategy, impurity, etc.   All params.

Reply:
Previously, I thought part of the parameters were enough for testing. But this morning, I tried adding featuresSubsetStrategy and found an issue that the fitMultiple or fit didn't update the value that should be mapped.

Really good suggestions.